content/en/tracing/guide/resource_monitor.md

```diff
@@ -1,44 +0,0 @@
----
-title: APM resource monitors
-kind: documentation
-aliases:
```

Comment:
When a page is deleted, it needs to be aliased. Is the information located elsewhere or is there a good page to alias?